game/src/Scope/GoldScoping.gd

```diff
@@ -0,0 +1,10 @@
+extends Label
+
+var instructions = "This is gold you have\n " \
```

Comment:
This is the gold you have:
Killing enemies drop gold and
Building towers cost gold

That's all for now. Have fun!

Missing spaces, capitalisation, and grammar fix.